popupStyle.css

```diff
@@ -0,0 +1,207 @@
+.popupContainer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 5;
+  opacity: 75;
+}
+
+.projectPopup {
+  display: flex;
+  flex-direction: column;
+  padding: 24px;
+  position: relative;
+  width: 75%;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, 5%);
+  z-index: 10;
+  overflow-y: scroll;
+  height: 700px;
+}
+
+.popupHeader {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.popupTitle {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: -5px;
+}
+
+.popTags {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  gap: 16px;
+  margin-top: -15px;
+}
+
+.tagsList1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.tagsList2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  list-style-type: square;
+}
+
+.tagsList3 {
+  list-style-type: square;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.popupBody {
+  width: 100%;
+}
+
+.popupImgContainer {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.popupImgContainer img {
+  width: 100%;
+  object-fit: contain;
+}
+
+.lowerContainer {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.popupTextContainer {
+  width: 100%;
+}
+
+.popupDescription {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.popupLinkContainer {
+  /* padding-left: 24px; */
+}
+
+.popupLanguages {
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  gap: 10px;
+  padding-left: 0;
+}
```

Comment:
- [OPTIONAL ] Kindly consider styling the languages to match that of the `Figma` design template in terms of the `background` and `text` color. Always make sure your design matches with provided design in any project you're building. Attention to detail is a very important skill all developers should have. See below for more details:
Figma:
<img width="580" alt="Screenshot 2023-04-22 at 13 59 16" src="https://user-images.githubusercontent.com/107192076/233789374-eee8884f-9ae7-4ec8-b902-67bfff5ed022.png">

Your design: 
<img width="344" alt="Screenshot 2023-04-22 at 14 00 49" src="https://user-images.githubusercontent.com/107192076/233789399-eb1aec24-44f7-4ff7-8deb-e44e201d095a.png">